ui/src/shared/data/queryFills.hson

```diff
@@ -0,0 +1,9 @@
+[
+  {type: 'null', text: 'null', menuOption: '(null)', inputValue: 'null', isValidForKapaNodes: true},
+  {type: 'previous', text: 'previous', menuOption: '(previous)', inputValue: 'previous', isValidForKapaNodes: false},
+  {type: 'number', text: 'number', menuOption: '(number)', inputValue: '0', isValidForKapaNodes: true},
+  {type: 'none', text: 'none', menuOption: '(none)', inputValue: 'none', isValidForKapaNodes: true},
+  {type: 'linear', text: 'linear', menuOption: '(linear)', inputValue: 'linear', isValidForKapaNodes: false},
+]
```

Comment:
Probably should use the `queryFillOptions` constants for the `type` field on these objects.  Would have to change this from an `hson` file to `js`, but no big deal!

Reply:
It's an interesting idea, but then the `type` wouldn't end up being useful in `FillQuery`, and matching would have to be done on `text` anyway since the string value for `NULL_STRING` is `null`. I just now tried making these changes and ended up throwing them away and leaving as-is.